app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if parts[0] == "cp" and len(parts) == 3:
+        source_file = parts[1]
+        destination_file = parts[2]
+
+        if source_file == destination_file:
```

Comment:
The condition to check if the source and destination files are the same is correct. However, ensure that this logic is clearly documented or commented for better code readability.